### DISCUSSION TITLE
To import old report on superset

### DISCUSSION THREAD

**Discussion by jayrajk112** [Sunbird-Obsrv/Community#125] [Issues] [answered]
@team

I want to know, how can i import  old reports created on superset. 
This is because just now we have reconfigure superset in our new environment  and hence we want to import old reports that were created in earlier environment .
Kindly help me in this task.

Thanks,
Jayraj Kakde

**Comment by manjudr** [comment, marked as answer]
@jayrajk112  - Are you referring to charts? Superset offers the capability to both export and import dashboards or charts. You can export a dashboard and then import it into the system wherever you need it.

**Comment by jayrajk112** [comment]
Thanks for reply
